### DISCUSSION TITLE
Push notifications are visible when computers is locked.

### DISCUSSION THREAD

**Discussion by latvianization** [linuxdeepin/developer-center#3129] [Features Request & Ideas | 特性请求 & 头脑风暴]
Push notifications are visible when computers is locked.

I lock my work computer but push notifications are appearing on lock screen.

**Comment by BLumia** [comment]
``` shell
$ sleep 10 && notify-send test
```

Can be reproduced under deepin v20.5 with the above command, when executed, lock the screen within 10 second and you'll see the notification on the lock screen.

I have confirmed this behavior is the designed behavior atm, so I'll mark this as a feature request and see if we plan to change it or imporve it in the future :)

**Reply by latvianization** [reply]
I used Deepin for work and this is one of the reason I switched to Zorin. Its not okay to show push notifications on locked computer. They can contain sensitive information

**Reply by BLumia** [reply]
I agree with you, but it's still the current designed behavior from the product manager's point of view, so I guess mark it as feature request make scene, and hopefully it will be changed in the future :)

**Comment by meiyixiang** [comment]
Hello, I have recorded this problem and fed back to relevant products
